### DISCUSSION TITLE
Roll back to the 1.10.1 release

### DISCUSSION THREAD

**Discussion by nayyala** [st3v3nmw/obsidian-spaced-repetition#761] [General]
It seems clear that there is more testing that needs to be done on the newest release of the plug in. Please consider rolling back to the earlier stable release, as like myself, I imagine many of us have found the bugs to be significant enough to interrupt our workflows

@st3v3nmw @ronzulu

**Comment by ronzulu** [comment]
Hi  @nayyala

Apart from https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/759 and https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/760, are there any other issues?

I expect to fix these tomorrow.

Cheers
Ronny

**Comment by 4Source** [comment]
For such cases I have to do some self-promotion I have published a plugin that allowed to install selected versions of other plugins. If there should be such a situation again, you could test [VARE](https://obsidian.md/plugins?id=vare).
